keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/README.md

```diff
@@ -351,7 +354,7 @@ For Beta and GA, add links to added tests together with links to k8s-triage for
 https://storage.googleapis.com/k8s-triage/index.html
 -->
 
-- <test>: <link to test coverage>
+No new integration tests for kubelet are planned.
```

Comment:
we don't have integration tests in the kubelet for resource managers (and perhaps in general), we use a combination of unit and e2e tests. Again probably unnecessary addition but won't hurt spelling it out.